sdk/python/tests/performance_tests.py

```diff
@@ -258,7 +258,7 @@ def load_pipeline_scripts() -> [(Path, str)]:
 
         path = path_name_dict["path"]
         name = path_name_dict.get("name") or Path(path).stem
-
+        copies = path_name_dict.get("copies")
```

Comment:
Maybe have a default value so it at least run once?
```suggestion
        copies = path_name_dict.get("copies", 1) 
```